gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/server/GremlinDriverIntegrateTest.java

```diff
@@ -1879,4 +1879,98 @@ public void shouldNotHangWhenSameRequestIdIsUsed() throws Exception {
 
         assertEquals(100, result1.get().one().getInt());
     }
+
+    @Test
+    public void shouldFailOnInitiallyDeadHost() throws Exception {
+        // Stop server
+        this.stopServer();
+
+        final Cluster cluster = TestClientFactory.build().create();
+        final Client client = cluster.connect();
+
+        try {
+            // try to re-issue a request now that the server is down
+            System.out.println("SUBMIT 1");
+            client.submit("g").all().get(3000, TimeUnit.MILLISECONDS);
```

Comment:
Don't need 3 second timeout for a local connection, we know the server isn't there anyway. how about 500 ms?

Also remove printout of SUBMIT 1 - it doesn't add much value to know.